builtin/bisect--helper.c

```diff
@@ -369,6 +419,8 @@ int cmd_bisect__helper(int argc, const char **argv, const char *prefix)
 			 N_("check and set terms in a bisection state"), CHECK_AND_SET_TERMS),
 		OPT_CMDMODE(0, "bisect-next-check", &cmdmode,
 			 N_("check whether bad or good terms exist"), BISECT_NEXT_CHECK),
+		OPT_CMDMODE(0, "bisect-terms", &cmdmode,
+			 N_("print out the bisect terms"), BISECT_TERMS),
```

Comment:
In the original commit, this is accompanied by a corresponding usage line: https://github.com/gitgitgadget/git/commit/db7b281c8ceb8d0662b4dc35edcd3422965b2500#diff-f42cbfc70e72a333b6ef20f77bf50af0R26. This line should probably also be added back into this commit.